packages/snap-preact/components/src/components/Organisms/Facet/Facet.tsx

```diff
@@ -487,6 +514,9 @@ interface OptionalFacetProps extends ComponentProps {
 	disableOverflow?: boolean;
 	previewOnFocus?: boolean;
 	valueProps?: any;
+	showSelectedCount?: boolean;
+	clearAllText?: string;
```

Comment:
Having the text determine the display of this button makes it so we cannot have default translations (similar to other props we have) - the best way to deal with this is to also have a hide/show prop. An example of this is the `label` prop in the SortBy component.